### ISSUE TITLE
backend openDAL Backend cannot be loaded: scheme is not enabled or supported

### ISSUE THREAD

**Issue by nickchomey** [rustic-rs/rustic#1161]
I am getting this error when I try to use onedrive and gdrive. I assume it is the case for others too..

`backend openDAL Backend cannot be loaded: Unsupported (permanent) at , context: { scheme: onedrive } => scheme is not enabled or supported`

```[repository]
repository = "opendal:onedrive"
password = ""

[repository.options]
access_token = "token"
root = "/root/path"
```
Here's what OpenDAL says about onedrive
https://docs.rs/opendal/latest/opendal/services/struct.Onedrive.html

When I change it to `opendal:gdrive` i get the same thing. 

But if I try b2 or s3, it just gives me an error regarding the bucket being invalid (since I didnt provide any credentials for it). That suggests to me that onedrive, gdrive and others are simply broken - either within OpenDAL itself or, more likely, how it was integrated into Rustic

**Comment by intgr** [comment]
I tried patching in OpenDAL OneDrive backend and using it. But there was no documentation about how to get the `access_token` value needed for it. After prompting them, here is some guidance how to do it, but it requires using some other library:

* https://github.com/apache/opendal/issues/4715#issuecomment-2157753501

Then I tried using GNOME's built-in OneDrive support via FUSE. This failed because there are memory leaks :laughing: reported here: https://gitlab.gnome.org/GNOME/gvfs/-/issues/742

**Comment by intgr** [comment]
The saga continues. I eventually got FUSE/GVFS/OneDrive backup done, by doing it chunk by chunk and restarting GVFS daemon before it leaked too much. But now I have repository corruption that cannot be resolved with `rustic repair` 😆

TL;DR: I found that **`rclone` is a much better way to work with OneDrive**, it is wayy faster than GVFS was. Although I think I have to redo the backup from scratch due to pre-existing corruption.

The documentation for rclone was a bit scarce, but I needed to run `rclone config` and add `repository = "rclone:<rclone-remote-name>:<relative-path>"`

More info on rclone OneDrive support here: https://rclone.org/onedrive/

**Comment by aawsome** [comment]
@nickchomey We are trying to prepare a patch version 0.8.1 soon and I think we can integrate the fix there.

**Comment by intgr** [comment]
Yes, rclone works great.

**Comment by aawsome** [comment]
@nickchomey Thanks for your suggestion. 
In fact, out of the 60 services, there are many which most likely will never be interesting as backup storage. And for those we actually don't want to compile support as that would just bloat the binary sizes without value

Moreover, as opendal designed the services as opt-in, there is no easy way to just just use all or maybe only opt-out non-needed services. Acuatally even in their self-provided CLI `oli` they specify each service directly, see https://github.com/apache/opendal/blob/main/bin/oli/Cargo.toml
